src/scripts/repl_server.py

```diff
@@ -15,18 +15,41 @@
 __already_loaded = False
 __ctx = {'__importlib': __importlib}
 
+class INST:
+    # Informs that it is not a supported instruction.
+    UNKNOWN = 0x00
+    # Send from server to client. Informs the client to print data.
+    PRINT = 0x01
+    # Send from client to server. Informs the REPL server that the executable .pyc file has been written out and is ready for evaluation.
+    LOAD = 0x02
+    # Send from server to client. Represents an exception.
+    EXCEPTION = 0x03
+    # Send from server to client. Tells the code generator to initialize due to an error.
+    INITIALIZE = 0x04
+    # Informs that the connection is to be / should be terminated.
+    EXIT = 0x05
+
+def __encode(instr, data=''):
+    data_bytes = data.encode()
+    data_len = len(data_bytes)
+    # one byte for inst, two bytes for size, and n bytes for data(Optional)
+    return instr.to_bytes(1, 'big') + data_len.to_bytes(2, 'big') + data_bytes
+
 while True:
     try:
-        __order = __client_socket.recv(1024).decode()
+        __data = __client_socket.recv(1024)
```

Comment:
I think there is no need to use double underscores anymore since exec is already in a separate namespace.

Reply:
Yes, you're right.